skyportal/tests/api/test_telescope.py

```diff
@@ -33,6 +33,36 @@ def test_token_user_post_get_telescope(upload_data_token, public_group):
             assert data['data'][key] == post_data[key]
 
 
+def test_fetch_telescope_by_name(upload_data_token, public_group):
+    name = str(uuid.uuid4())
+    post_data = {'name': name,
```

Comment:
I saw `id` fields in some of the seed configs (in one of Danny's recent PRs, I think), do you know why they ended up there and were they used in any way?

Reply:
no idea!